test/stress/host/agent_config.json

```diff
@@ -0,0 +1,98 @@
+{
+    "agent": {
+        "debug": true
+    },
+    "metrics": {
+      "namespace": "CloudWatchAgentStress",
+      "metrics_collected": {
+        "cpu": {
+          "measurement": [
+            "time_active", "time_guest", "time_guest_nice", "time_idle", "time_iowait", "time_irq",
+            "time_nice", "time_softirq", "time_steal", "time_system", "time_user",
+            "usage_active", "usage_guest", "usage_guest_nice", "usage_idle", "usage_iowait", "usage_irq",
+            "usage_nice", "usage_softirq", "usage_steal", "usage_system", "usage_user"
+          ],
+          "metrics_collection_interval": 1
+        },
+        "swap": {
+            "measurement": [
+                "free","used","used_percent"
+            ]
+        },
+        "processes": {
+            "measurement": [
+              "blocked","running","sleeping","stopped","total","dead","idle","paging","total_threads","zombies"
+            ],
+            "metrics_collection_interval": 1
+        },
+        "netstat": {
+            "measurement": [
+              "tcp_close","tcp_close_wait","tcp_closing", "tcp_established","tcp_fin_wait1","tcp_fin_wait2","tcp_last_ack",
+              "tcp_listen","tcp_none","tcp_syn_sent","tcp_syn_recv","tcp_time_wait","udp_socket"
+            ],
+            "metrics_collection_interval": 1
+          },
+          "mem": {
+            "measurement": [
+              "active", "available", "available_percent", "buffered", "cached", "free", "inactive", "total",
+              "used", "used_percent"
+            ],
+            "metrics_collection_interval": 1
+          },
+          "diskio": {
+            "resources": [
+              "*"
+            ],
+            "measurement": [
+              "iops_in_progress", "io_time", "reads", "read_bytes", "read_time", "writes", "write_bytes", "write_time"
+            ],
+            "metrics_collection_interval": 1
```

Comment:
For `ethtool` it does not support `metric_collection_interval` since [being disabled at JSON level](https://github.com/khanhntd/amazon-cloudwatch-agent/blob/main/translator/config/schema.json#L466). However, IMO, it should be enable since its a [Regular Input](https://github.com/influxdata/telegraf/blob/6a3b27126a26dd43d2a287b4c9d8831dca6bf687/plugins/inputs/ethtool/ethtool_linux.go#L60-L88) (collecting metrics every X second) instead of a Service Input (open a server and gather metrics being sent - prometheus) so it always use `agent_collection_interval`; not sure why we disable it though.

Reply:
no strong desire, resolving.